_apps/remove-outside-collaborators.md

```diff
@@ -0,0 +1,78 @@
+---
+# A human-friendly name of your listing
+title: Remove Outside Collaborators
+# A short description of what your app does
+description: A GitHub Probot App that monitors and instantly removes any outside collaborators added to your Organization.
```

Comment:
All the apps on the website are Probot Apps, so we'd prefer this to be: 'Monitors and instantly removes any outside collaborators added to your organization.'